src/WooCommerce.php

```diff
@@ -26,6 +26,36 @@ public static function woocommerce_get_settings_shop_standards(array $settings):
       'type' => 'sectionend',
       'id' => Plugin::L10N,
     ];
+    $settings[] = [
+      'type' => 'title',
+      'name' => __('Automatic sale category assignment', Plugin::L10N),
+    ];
+    $settings[] = [
+      'type' => 'checkbox',
+      'id' => '_' . Plugin::L10N . '_enable_auto_sale_category_assignment',
+      'name' => __('Enable', Plugin::L10N),
+      'show_if_checked' => 'option',
+    ];
+    $settings[] = [
+      'type' => 'multiselect',
+      'id' => '_' . Plugin::L10N . '_eligible_delivery_times',
```

Comment:
@moritzbappert All settings related to a particular feature should have the same name prefix, and their names should describe more clearly what they are about; e.g.:

```
_sale_auto_label_enabled
_sale_auto_label_delivery_times
_sale_auto_label_category
```
A well-scoped feature like this should also be located in a separate class WooCommerceSaleLabel instead of being added to the pile of unknown adjustments in the WooCommerce class (including its settings fields).

Reply:
@sun Disregarding that it's already been merged, do you want me to create a new branch, rename the field IDs and organize this feature in its own class?